### PR TITLE
Bug 1141971 - [Privacy Controls] fix remaining words

### PR DESCRIPTION
**Pull request by yuwen41200**
Rename remaining 'privacy panel' to 'privacy controls'

### CODE REVIEW

File: apps/settings/locales/settings.en-US.properties [hunk 1]
```diff
@@ -955,7 +955,7 @@ clear=Clear
 
 # Security :: Privacy Panel
 privacy-controls=Privacy Controls
-no-privacy-panel=No privacy panel app found
+no-privacy-panel=No privacy controls app found
```
Comment (gasolin) [inline apps/settings/locales/settings.en-US.properties]: please change the naming from `no-privacy-panel` to `no-privacy-controls`, so the l10n community can detect the keyword change and made new translation

**Comment by gasolin** [comment]
nit: suggest change the commit message to `Rename remaining 'privacy panel' to 'privacy controls'`

**Comment by try-server-hook** [comment]
[yuwen41200 Yu-wen Pwu (yuwen41200)](https://github.com/yuwen41200) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=83923666e455)

**Comment by try-server-hook** [comment]
[yuwen41200 Yu-wen Pwu (yuwen41200)](https://github.com/yuwen41200) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7c75a3f8a282)

**Comment by autolander** [comment]
The pull request could not be integrated in as it contained a commit which was missing a bug number. Please ensure all commits contain bug numbers, and are in the format of: Bug [number] - [description]

**Pull request merged**